crates/shared/src/arguments.rs

```diff
@@ -1,12 +1,16 @@
 //! Contains command line arguments and related helpers that are shared between the binaries.
 use crate::{
+    fee_subsidy::cow_token::SubsidyTiers,
```

Comment:
I think this `struct Arguments` is also used by the driver but these arguments don't make sense for the driver. Can make new structs like `OrderCreationArguments` which would be only used by orderbook and autopilot.